templates/js-template-default/frameworks/runtime-src/proj.openharmony/entry/src/main/module.json5

```diff
@@ -5,7 +5,8 @@
     "description": "$string:entry_desc",
     "mainElement": "EntryAbility",
     "deviceTypes": [
-      "default"
+      "phone",
```

Comment:
Has the default type been removed? If there are other types in the future that need to be modified by users?

Also, what type is rk3568?